lib/internal/bootstrap_node.js

```diff
@@ -221,15 +221,52 @@
   }
 
   function setupGlobalConsole() {
+    var inspectorConsole;
+    var wrapConsoleCall;
+    if (process.inspector) {
+      inspectorConsole = global.console;
+      wrapConsoleCall = process.inspector.wrapConsoleCall;
+      delete process.inspector.wrapConsoleCall;
```

Comment:
I would prefer if `inspector` wasn't a public property on `process` (except during startup). Could you get away with deleting `process.inspector` here?